openassessment/xblock/data_conversion.py

```diff
@@ -255,6 +255,31 @@ def create_submission_dict_v2(submission, prompts, staff_view=False):
     return submission
 
 
+def update_submission_old_format_answer(submission):
+    """
+    Update the submission answer from indexed-key format to new format, the format that uses
+    semantically correct keys
+    """
+    answer = submission['answer']
+
+    if '0' in answer and '1' and answer:
```

Comment:
`and '1' and answer:` shouldn't this be `'1' in answer`

Reply:
Oh yeah.